chalice/app.py

```diff
@@ -350,6 +359,13 @@ def __eq__(self, other):
         return False
 
 
+class File(object):
```

Comment:
Is there a rationale for deviating from what flask has?  Seems like it would help if we use the same attribute names.

Also, I think we should at least add the content type and the filename to this object (if available), those are commonly used fields.

Reply:
What do you mean? Adding this header tells the email parse function how to parse the following content. The individual files/keys are unaffected.